src/zui/ZUICreatePerson/index.tsx

```diff
@@ -0,0 +1,200 @@
+import isEmail from 'validator/lib/isEmail';
+import isURL from 'validator/lib/isURL';
+import { isValidPhoneNumber } from 'libphonenumber-js';
+import {
+  Box,
+  Button,
+  CircularProgress,
+  Dialog,
+  Divider,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material';
+import { FC, MouseEvent, useState } from 'react';
+
+import messageIds from 'zui/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import PersonalInfoForm from './PersonalInfoForm';
+import useCreatePerson from 'features/profile/hooks/useCreatePerson';
+import useCustomFields from 'features/profile/hooks/useCustomFields';
+import { useNumericRouteParams } from 'core/hooks';
+import {
+  ZetkinCreatePerson,
+  ZetkinCustomField,
+  ZetkinPerson,
+} from 'utils/types/zetkin';
+
+interface ZUICreatePersonProps {
+  onClose: () => void;
+  onSubmit?: (e: MouseEvent<HTMLButtonElement>, person: ZetkinPerson) => void;
+  open: boolean;
+  title?: string;
+  submitLabel?: string;
+}
+
+const ZUICreatePerson: FC<ZUICreatePersonProps> = ({
+  open,
+  onClose,
+  onSubmit,
+  title,
+  submitLabel,
+}) => {
+  const theme = useTheme();
+  const { orgId } = useNumericRouteParams();
+  const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
+  const customFields = useCustomFields(orgId).data ?? [];
+  const createPerson = useCreatePerson(orgId);
+
+  const [tags, setTags] = useState<number[]>([]);
+
+  const [personalInfo, setPersonalInfo] = useState<ZetkinCreatePerson>({});
+
+  return (
+    <Dialog
+      fullScreen={fullScreen}
+      fullWidth
+      onClose={() => {
+        onClose();
+        setPersonalInfo({});
+        setTags([]);
+      }}
+      open={open}
+    >
+      <Box sx={{ padding: '40px 0 40px 40px' }}>
+        <Box display="flex">
+          <Typography mb={2} variant="h5">
+            <Msg id={messageIds.createPerson.title.default} />
+          </Typography>
+          <Typography sx={{ ml: 0.5 }} variant="h5">
+            {title}
+          </Typography>
```

Comment:
Concatenating strings like this does not work with internationalization. Just do this instead:

```suggestion
          <Typography sx={{ ml: 0.5 }} variant="h5">
            {title}
          </Typography>
```

And make sure that the various call sites for this component pass in a message that includes the full title.